### DISCUSSION TITLE
（NUXTJS） Only configuring one network, no response when clicked

### DISCUSSION THREAD

**Discussion by whaleshi** [WalletConnect/walletconnect-monorepo#5525] [Developer Support] [answered]
![WX20241128-180619](https://github.com/user-attachments/assets/e124f296-c37c-45e7-9534-e641060ea041)
![WX20241128-180631](https://github.com/user-attachments/assets/538adbfa-4ad5-454d-8848-0a9876db6fab)
Metamask is not on 56, but there is no response when clicking to switch networks

**Comment by arein** [comment]
which version of AppKit are you on and what is the expected behavior @pawnJj ?

**Reply by whaleshi** [reply]
1.5.0
At this moment, my network is in ETH. I want to click on BNB Smart Chain to trigger Metamask to switch networks, but it doesn't

**Reply by whaleshi** [reply]
In NUXTJS, useAppKitNetwork will set the first network in networks as the default and then click on it without any response

**Reply by arein** [reply]
I believe we just fixed this issue. Can you pls upgrade to v1.5.2?

**Reply by whaleshi** [reply]
I'll try

**Reply by whaleshi** [reply]
Still can't do it @arein

**Reply by whaleshi** [reply]
You can click now, but it keeps loading after the switch is successful
![WX20241128-192540](https://github.com/user-attachments/assets/c1984efc-b00b-4cf5-945c-56b998c3e89f)

**Reply by arein** [reply]
@svenvoskamp is on it! thanks for reporting @pawnJj - stay tuned!

**Reply by whaleshi** [reply]
When networks have two or more networks, loading will disappear

**Reply by whaleshi** [reply]
I'm quite anxious about how long it will take to complete. Thank you very much @arein

**Reply by arein** [reply]
@svenvoskamp is on it - should be released shortly

**Reply by whaleshi** [reply]
Thank you. After the release, please let me know the version number @arein

**Reply by whaleshi** [reply, marked as answer]
1.5.3
